### PR TITLE
Add docstring to facilitate doc generation for dockerize subcommand

### PR DESCRIPTION
**Pull request by ishaileshpant**

### CODE REVIEW

File: openfl/interface/workspace.py [hunk 1]
```diff
@@ -402,8 +402,34 @@ def export_() -> str:
...
+        context: The context object that provides access to shared information and methods,such as invoking other commands or accessing configuration settings.
+        save (bool): If True, saves the workspace Docker image as a tarball (`.tar`) file.
+        rebuild (bool): If True, forces a rebuild of the Docker images without using the cache.
+        revision (str): Specifies the OpenFL revision to use as a build argument in the Docker build process.
+
+    Returns:
+        None
 
+    Raises:
+        RuntimeError: If any of the Docker commands fail during execution.
```
Comment (MasterSkepticista) [inline openfl/interface/workspace.py]: This portion isn't required, click generates argument-specific docstrings from the options decorators.

File: openfl/interface/workspace.py [hunk 2]
```diff
@@ -402,8 +402,34 @@ def export_() -> str:
...
+
+    Example:
+        To package the workspace and save the Docker image as a tarball:
+
+        ```python
+        dockerize_(context, save=True, rebuild=False, revision='main')
```
Comment (MasterSkepticista) [inline openfl/interface/workspace.py]: Users won't be calling the python functions, so I suggest providing command examples.

**Comment by MasterSkepticista** [comment]
@ishaileshpant With #1140 merged, this [readme](https://github.com/securefederatedai/openfl/blob/develop/openfl-docker/README.md) provides (incomplete) details on how to use dockerize subcommand. I am open to ideas on how (and where) to best document practices on how to run an enclave-enabled container.

**Comment by ishaileshpant** [comment]
> @ishaileshpant With #1140 merged, this [readme](https://github.com/securefederatedai/openfl/blob/develop/openfl-docker/README.md) provides (incomplete) details on how to use dockerize subcommand. I am open to ideas on how (and where) to best document practices on how to run an enclave-enabled container.

May be first thing is to fix this readme or link a doc specifically detailing enclave based run possibly here https://github.com/securefederatedai/openfl/tree/develop/docs/developer_guide/advanced_topics - wdyt @MasterSkepticista ?

File: openfl/interface/workspace.py (hunk 2 above)
Comment (ishaileshpant) [reply openfl/interface/workspace.py]: I've removed the python example altogether till we figure out best way to document all the workspace (or any such command wrapper) that is used with openfl binary wdyt @MasterSkepticista ?

**Comment by ishaileshpant** [comment]
Closing this PR as more details will follow on documentation as per the governance document that @MasterSkepticista is working on - We want to take on a broader effort on documentation and dev doc generation automation

**Pull request closed without merging**